Packs/FeedDHS/ReleaseNotes/2_0_15.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### DHS Feed v2
+
+Fixed an issue in the **TAXII2ApiModule** script.
```

Comment:
Is this using the TAXII2ApiModule? Does it use the authentication headers?

Reply:
no